src/cubical/Cof.mli

```diff
@@ -26,6 +27,12 @@ val join : ('a, 'v) cof list -> ('a, 'v) cof
 (** Given a list [φ0,...,φn : 𝔽] of cofibrations, the conjunction [φ0 ∧ ... ∧ φn]. *)
 val meet : ('a, 'v) cof list -> ('a, 'v) cof
 
+(** Given dimensions [r, r' : 𝕀], a cofibration equivalent to [¬(r=r')] that does not use [¬]. *)
+val neg_eq : dim0:'a -> dim1:'a -> 'a -> 'a -> ('a, 'v) cof
```

Comment:
It turns out we only need the instances where `'a = Dim.dim`. This means `dim0 is always Dim.Dim0` and `dim1` is always `Dim.Dim1`. Polymorphism is useless but could be aesthetically nice to keep.